x-pack/docs/en/security/securing-communications/security-basic-setup-https.asciidoc

```diff
@@ -0,0 +1,595 @@
+| `metricbeat_reader` | Read {metricbeat} data.
+| `monitoring_user`   | Allow users to monitor the health of {metricbeat}
+itself. Only assign this role to users who manage {metricbeat}
+| `beats_admin`       | Create and manage configurations in {beats} central
+management. Only assign this role to users who need to use {beats} central
+management.
+|===
+
+**Next**: <<configure-metricbeat-tls,Configure {metricbeat} to use TLS>>
+
+[discrete]
+[[configure-metricbeat-tls]]
+===== Configure {metricbeat} to use TLS
+
+Before starting {metricbeat}, you configure the connections to {es} and
+Kibana. You can configure authentication to send data to your secured cluster
+using basic authentication, API key authentication, or Public Key
+Infrastructure (PKI) certificates.
+
+The following instructions use the credentials for the `metricbeat_writer`
+and `metricbeat_setup` roles that you created. If you need a greater level of
+security, we recommend using PKI certificates.
+
+After configuring connections to Elasticsearch and Kibana, you'll enable the
+`elasticsearch-xpack` module and configure that module to use HTTPS.
+
+WARNING: In production environments, we strongly recommend using a separate
+cluster (referred to as the monitoring cluster) to store your data. Using a
+separate monitoring cluster prevents production cluster outages from impacting
+your ability to access your monitoring data. It also prevents monitoring
+activities from impacting the performance of your production cluster.
+
+. From the directory where you installed Elasticsearch, navigate to the
+`/kibana` directory that you created when <<encrypt-http-communication,encrypting HTTP client communications for {es}>>.
+
+. Copy the `elasticsearch-ca.pem` certificate to the directory where you
+installed Metricbeat.
+
+. Open the `metricbeat.yml` configuration file and configure the connection
+to Elasticsearch.
++
+Under `output.elasticsearch`, specify the following fields:
++
+[source,yaml]
+----
+output.elasticsearch:
+ hosts: ["<your_elasticsearch_host>:9200"]
+ protocol: "https"
+ username: "metricbeat_writer"
+ password: "<password>"
+ ssl:
+   certificate_authorities: ["elasticsearch-ca.pem"]
+   verification_mode: "certificate"
+----
+
+   `hosts`:: Specifies the host where your Elasticsearch cluster is running.
+
+   `protocol`:: Indicates the protocol to use when connecting to Elasticsearch.
+   This value must be `https`.
+
+   `username`:: Name of the user with privileges required to publish events to
+   Elasticsearch. The `metricbeat_writer` role that you created has these
+   privileges.
+
+   `password`:: Password for the indicated `username`.
+
+   `certificate_authorities`:: Indicates the path to your trusted CA.
+
+. Configure the connection to Kibana.
++
+Under `setup.kibana`, specify the following fields:
++
+[source,yaml]
+----
+setup.kibana
+ host: "https://<your_elasticsearch_host>:5601"
+ ssl.enabled: true
+ username: "metricbeat_setup"
+ password: "p@ssw0rd"
```

Comment:
Unless kibana's server certificate is signed by a public CA or centrally managed CA, we probably need a CA setting here?